docs/alerting/email-destination.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Destination type - Email
+nav_order: 1
+parent: monitors
+has_children: false
+---
+
+#  Email as a Destination 
+A destination is a location where you can send an alert that was triggered by the monitor. To send the alert as an email, select **Email** as the destination **Type**. You would need to configure at least one  **Sender** email address and add a **Recipient** email address. These two settings can be configured and managed using **Manage senders** and **Manage email groups**. 
```

Comment:
Lead with a use case, not a definition. "If you want to receive a notification by email when an alert triggers..." or "Using the email destination lets you..."

Reply:
Done